tests/baselines/reference/completionsClassMembers4.baseline

```diff
@@ -0,0 +1,254 @@
+            {
+              "text": "async",
+              "kind": "keyword"
+            }
+          ]
+        },
+        {
+          "name": "constructor",
+          "kind": "keyword",
+          "kindModifiers": "",
+          "sortText": "15",
+          "displayParts": [
+            {
+              "text": "constructor",
+              "kind": "keyword"
+            }
+          ]
+        },
+        {
+          "name": "declare",
+          "kind": "keyword",
+          "kindModifiers": "",
+          "sortText": "15",
+          "displayParts": [
+            {
+              "text": "declare",
+              "kind": "keyword"
+            }
+          ]
+        },
+        {
+          "name": "get",
+          "kind": "keyword",
+          "kindModifiers": "",
+          "sortText": "15",
+          "displayParts": [
+            {
+              "text": "get",
+              "kind": "keyword"
+            }
+          ]
+        },
+        {
+          "name": "override",
+          "kind": "keyword",
+          "kindModifiers": "",
+          "sortText": "15",
+          "displayParts": [
+            {
+              "text": "override",
+              "kind": "keyword"
+            }
+          ]
+        },
+        {
+          "name": "private",
+          "kind": "keyword",
+          "kindModifiers": "",
+          "sortText": "15",
+          "displayParts": [
+            {
+              "text": "private",
+              "kind": "keyword"
+            }
+          ]
+        },
+        {
+          "name": "protected",
+          "kind": "keyword",
+          "kindModifiers": "",
+          "sortText": "15",
+          "displayParts": [
+            {
+              "text": "protected",
+              "kind": "keyword"
+            }
+          ]
+        },
+        {
+          "name": "public",
+          "kind": "keyword",
+          "kindModifiers": "",
+          "sortText": "15",
+          "displayParts": [
+            {
+              "text": "public",
+              "kind": "keyword"
+            }
+          ]
+        },
+        {
+          "name": "readonly",
+          "kind": "keyword",
+          "kindModifiers": "",
+          "sortText": "15",
+          "displayParts": [
+            {
+              "text": "readonly",
+              "kind": "keyword"
+            }
+          ]
+        },
+        {
+          "name": "set",
+          "kind": "keyword",
+          "kindModifiers": "",
+          "sortText": "15",
+          "displayParts": [
+            {
+              "text": "set",
+              "kind": "keyword"
+            }
+          ]
+        },
+        {
+          "name": "static",
+          "kind": "keyword",
+          "kindModifiers": "",
+          "sortText": "15",
+          "displayParts": [
+            {
+              "text": "static",
+              "kind": "keyword"
+            }
+          ]
+        },
+        {
+          "name": "SOME_TEXT",
```

Comment:
@DanielRosenwasser Do we need to add brackets for computed (`["SOME_TEXT"]`, `[SOME_CONSTANT]`?) inherited members? [Playground](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBAYwgOwM7wMoHkCyBRAfQGEsA5DAFQEFSK4BeOAcm3wIrwA0KmBuAKFCRYiADYBDVKjgAhScDgBvfnFVwA2q0Ily1WgF0AXHABGECKODjkDOADNxo1MAEBfQeGjwEEqXAAiwFAAlgBuwAAmcKAwwMgR0nLOSipq-O5AA)

Reply:
Probably better to write `[SOME_CONSTANT]` if it's reasonable. `["SOME_TEXT"]` is a good alternative.